src/main/java/io/tiledb/cloud/TileDBSQL.java

```diff
@@ -48,29 +49,39 @@ public TileDBSQL(TileDBClient tileDBClient, String namespace, SQLParameters sql)
 
     /**
      * Exec an SQL query and get results in arrow format.
+     *
+     * @return
```

Comment:
Can you add a comment on this return type?

Reply:
Done!